src/main/webapp/content/scss/global.scss

```diff
@@ -504,3 +504,17 @@ code.password:hover {
 .cdk-overlay-container {
     z-index: 1060;
 }
+
+/* ==========================================================================
+Course Info Bar
+========================================================================== */
+.course-info-bar {
```

Comment:
I'm not sure about putting this in the global.scss file

Reply:
Where would you put it for making it available in all the application?

Reply:
if we reuse CSS it is maybe useful to create a component for the course-info-bar with an component based css. 
This is just my opinion. I'm curious what @krusche and @jpbernius do think about this

Reply:
Most of the style is actually in a global (s)css file. So I guess it is ok like that.

Reply:
then it is fine for me as well